homeassistant/components/tfl/config_flow.py

```diff
@@ -0,0 +1,230 @@
+"""Config flow for Transport for London integration."""
+from __future__ import annotations
+
+from copy import deepcopy
+import logging
+from typing import Any
+from urllib.error import HTTPError, URLError
+
+from tflwrapper import stopPoint
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+import homeassistant.helpers.config_validation as cv
+
+from .config_helper import config_from_entry
+from .const import (
+    CONF_API_APP_KEY,
+    CONF_STOP_POINT,
+    CONF_STOP_POINT_ADD_ANOTHER,
+    CONF_STOP_POINTS,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_API_APP_KEY, default=""): cv.string,
+    }
+)
+STEP_STOP_POINT_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_STOP_POINT): cv.string,
+        vol.Optional(CONF_STOP_POINT_ADD_ANOTHER): cv.boolean,
+    }
+)
```

Comment:
I agree with the comment from @joostlek, a select selector allowing the user to add multiple stops may work better.

Reply:
Done